src/__tests__/test-utils/render_with_extension.tsx

```diff
@@ -1,58 +1,50 @@
 /*
-
  MIT License
-
- Copyright (c) 2020 Looker Data Sciences, Inc.
-
+ Copyright (c) 2021 Looker Data Sciences, Inc.
  Permission is hereby granted, free of charge, to any person obtaining a copy
  of this software and associated documentation files (the "Software"), to deal
  in the Software without restriction, including without limitation the rights
  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
  copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions:
-
  The above copyright notice and this permission notice shall be included in all
  copies or substantial portions of the Software.
-
  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  SOFTWARE.
-
  */
 
-import React from 'react';
-import { DiagramToolbar } from '../../components/DiagramFrame/DiagramCanvas/DiagramToolbar'
-import { shallow } from 'enzyme';
-import 'jest-styled-components'
-
-// jest.mock("@looker/components", () => ({
-//   Divider: () => "Divider",
-//   Flex: () => "Flex",
-//   FlexItem: () => "FlexItem",
-//   Card: () => "Card",
-//   IconButton: () => "IconButton",
-//   theme: {colors: {key:"rgb(45, 126, 234)"}, space: {large: "2em"}, fontSizes: {large: "2em"}, fontWeights: {normal: "1em"}}
-// }))
-
-describe('<DiagramToolbar />', () => {
-  const basic = shallow(
-  <DiagramToolbar
-    zoomFactor={1}
-    reload={false}
-    defaultMinimap={true}
-    minimapEnabled={true}
-    minimapUntoggled={true}
-    setZoomFactor={undefined}
-    setViewPosition={undefined}
-    setReload={undefined}
-    setMinimapUntoggled={undefined}
-    setMinimapEnabled={undefined}
-  />);
-  it('should match the basic', () => {
-    expect(basic.debug()).toMatchSnapshot();
-  });
-});
+import React, { ReactElement } from 'react'
+import { renderWithTheme } from '@looker/components-test-utils'
+import {  ExtensionContext2, ExtensionContextData2 } from "@looker/extension-sdk-react"
+import {  LookerHostData,  registerHostApi, ExtensionHostApi } from "@looker/extension-sdk"
+import { Looker40SDK } from '@looker/sdk'
+
+const withExtensionContext = (
```

Comment:
This is okay for now. should consider moving this as a test util to sdk codegen exported possibly from extension sdl react.